specification/applicationinsights/resource-manager/readme.md

```diff
@@ -274,6 +274,7 @@ These settings apply only when `--tag=package-2018-06-17-preview` is specified o
 
 ``` yaml $(tag) == 'package-2018-06-17-preview'
 input-file:
+- Microsoft.Insights/preview/2018-06-17-preview/workbookOperations_API.json
```

Comment:
This is up to the owner team to include, depending on what they want to include in the specific package. Tagging @ericc1103 who introduced this API version to check whether this should be included here or not.

Reply:
Adding @jgardner04 to discuss this together.